package.json

```diff
@@ -229,8 +229,9 @@
     "@metamask/controller-utils": "^1.0.0",
     "@metamask/design-tokens": "^1.9.0",
     "@metamask/desktop": "^0.3.0",
-    "@metamask/eth-json-rpc-infura": "^7.0.0",
-    "@metamask/eth-json-rpc-middleware": "^10.0.0",
+    "@metamask/eth-json-rpc-infura": "8.0.0",
```

Comment:
Can we use `^` here?

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/17652/commits/4e3911a00695766a0edef23a4958edc66779bb24